float64 xGoal
float64 yGoal
---
int64 status